README.md

```diff
@@ -39,14 +47,15 @@ npm run start
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](https://github.com/MahdiAghaali/{PROJECTLINK}/issues).
+Feel free to check the [issues page](https://github.com/MahdiAghaali/air-pollution-metrics/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
 - Hat tip to anyone whose code was used
 - Inspiration
 
```

Comment:
Great job @MahdiAghaali 🤝

- [ ] [Optional] Kindly fix your MIT license link which appears broken at the moment. This will make your readme file more professional.